requirement.txt

```diff
@@ -0,0 +1,13 @@
+appdirs==1.4.0
```

Comment:
you probably only want to add top level requirements such as pillow/numpy i.e. the ones you explicitly installed and not their dependencies